puppet/zulip/manifests/postgres_appdb_base.pp

```diff
@@ -12,10 +12,21 @@
         # Needed for our full text search system
         "${postgresql}-tsearch-extras",
       ]
-      zulip::safepackage {
+      $debian_release_version_array = scanf("${::operatingsystemrelease} ", '%d')
```

Comment:
You mentioned that you copied this trick from #12985, but you don’t need it here: `$::operatingsystemrelease` is of string type in all versions of facter, so `scanf($::operatingsystemrelease, '%d')` is fine.

Reply:
Thanks for letting me know. But since we will match the release name string instead, I won't be making this change (instead the line(s) will just be removed).